### ISSUE TITLE
Add Metered Wall Functionality

### ISSUE THREAD

**Issue by ksmoore125** [laterpay/laterpay-wordpress-plugin#1296]
# User Story
As a LaterPay merchant, I would like to be able to enable a metered wall so that I can follow the trends of the larger publishers in the industry.

# Acceptance Criteria
See additional discovery documentation [here](https://docs.google.com/document/d/1ajMHkH-OYtw_djPjKjy85QSPvUAr6EMegiLSWAEJHNg/edit#heading=h.imtwvqvo0hxy)
- Count articles that would normally be blocked by the LaterPay paywall (regardless of the price)
- As long as the number above (`x`) is less than or equal to the number configured on the LaterPay Settings page (https://github.com/laterpay/laterpay-wordpress-plugin/issues/1293), within the given time period, then display https://github.com/laterpay/laterpay-wordpress-plugin/issues/1295
- When the number `x` exceeds he number configured on the LaterPay Settings page (https://github.com/laterpay/laterpay-wordpress-plugin/issues/1293), within the given time period, then display https://github.com/laterpay/laterpay-wordpress-plugin/issues/1294
- Any articles which were read for free, should remain free to the user as long as they do not clear their cookies ([see discussion here](https://docs.google.com/a/laterpay.net/document/d/1ajMHkH-OYtw_djPjKjy85QSPvUAr6EMegiLSWAEJHNg/edit?disco=AAAADTxRSkI))

**Comment by ksmoore125** [comment]
Won't Do
